lib/engine/game/g_1822_nrs/game.rb

```diff
@@ -12,6 +12,7 @@ class Game < G1822::Game
         include_meta(G1822NRS::Meta)
         include Entities
         include Map
+        include G1822::Scenario
```

Comment:
I notice this file doesn't have `require_relative '../g_1822/scenario'` like the others do, does it still resolve the issue without adding it?

Reply:
Just tested locally, without the require line I still get the error, but with it the error is gone.

Reply:
Will merge and push this line separately.